pkg/amazonlinux2016.09/Dockerfile

```diff
@@ -137,11 +138,14 @@ CMD [ "pyinstaller --onedir --noconfirm --log-level ${_BINARY_LOG_LEVEL} --addit
     && fpm -s dir -t rpm \
        -n hubblestack \
        -v ${HUBBLE_VERSION} \
+       --iteration ${HUBBLE_ITERATION} \
     #todo: get rid of the git dependency with static bin in the future
        -d 'git' \
-       --config-files /etc/hubble --config-files /etc/osquery/osquery.conf \
+       --config-files --config-files /etc/osquery/osquery.conf \
```

Comment:
Why are we not marking /etc/hubble as config-file anymore?

Reply:
In DX, we want the ability to deliver new scheduled jobs without having to work with the users manually, to keep maintenance costs low. This is why we added the `/etc/hubble/hubble.d/*.conf support` -- we want to move all user config into those files so that  when the hubble package upgrades, it can upgrade `/etc/hubble/hubble` as well and won't clobber user config or be clobbered by user config.